10up/classifai
# #430: Use proper plurals based on image settings

### Description of the Change
Adds singular labels where a single image is selected for generation.

Closes #430

### How to test the Change

**Test 1: Plural**:
- Open Classifai settings:
  - set the number of images to generate between 2 - 10 (i.e. not 1)
  - Check / ensure: `Enable image generation` is checked
- Create a new post:
  - Open the media modal (in a new post => featured image / media block) and select `Generate Images`
  - Confirm the text remains as:
    - `Enter a prompt below to generate images.`
    - `Once images are generated, choose one or more of those to import into your Media Library and then choose one image to insert.`

**Test 3: Singular**:
- Open Classifai settings:
  - set the number of images to generate 1 (i.e. not 2 - 10)
  - Check / ensure: `Enable image generation` is checked
- Create a new post:
  - Open the media modal (in a new post => featured image / media block) and select `Generate Images`
  - Confirm the text shows singular terminology:
    - `Enter a prompt below to generate an image.` - new
    - `Once an image is generated, you can import it into your Media Library and then select to insert.` - new



### Changelog Entry
> Added - Add singular labels when a single image is selected for generation.

### Credits
Props @jamesmorrison


### Checklist:
